flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1570,7 +1573,7 @@ ContainerLaunchContext setupApplicationMasterContainer(
 			boolean hasLogback,
 			boolean hasLog4j,
 			boolean hasKrb5,
-			int jobManagerMemoryMb) {
+			ProcessMemorySpec processSpec) {
```

Comment:
Why not `JobManagerProcessSpec`? Do we have any use case that possibly pass in a different `ProcessMemorySpec` implementation?